Enhanced Log/enhancedsyslogsettings.txt

```diff
@@ -87,76 +87,77 @@ Other Highlight Color=Andere Hervorhebungsfarbe
 :end
 
 :enhanced_log_color_error_plug:
-> Color to use for 'Error' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Fehler'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_minor_plug:
-> Color to use for 'Minor Issues' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'kleine Unzulänglichkeiten'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_limetech_plug:
-> Color to use for 'Lime Tech' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Lime Tech'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_array_plug:
-> Color to use for 'Array' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Array'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_system_plug:
-> Color to use for 'System' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'System'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem a '#'.
 :end
 
 :enhanced_log_color_file_system_plug:
-> Color to use for 'File System' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Dateisystem'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem a '#'.
 :end
 
 :enhanced_log_color_drive_plug:
-> Color to use for 'Drive Related' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Datenträger'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_network_plug:
-> Color to use for 'Network' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Netzwerk'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_login_plug:
-> Color to use for 'Login' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Login'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
```

Comment:
@vakilando statt "Login" -> "Anmeldung"?

Reply:
Ja. 
Ich war/bin unsicher ob die Texte wie "error", "minor issues", "lime tech", "array", "system", "file system", "drive related", "network", "login", "emhttp", "other" so im Log auftauchen... Wenn ja, sollten sie nicht übersetzt werden....dachte ich.

Reply:
Lassen wir sie unübersetzt ist glaub ich beim log einfacher

Reply:
So, habe noch einmal Änderungen vorgenommen. 
Ich habe die Originalbezeichnungen  "error", "minor issues", "lime tech", "array", "system", "file system", "drive related", "network", "login", "emhttp", "other" meistens belassen und ggf eine deutsche Übersetzung in Klammern dahinter gesetzt.
Sp OK? Bitte prüfen.